index.js

```diff
@@ -13,63 +13,115 @@ const FAQ = [
     }
 ];
 
-// add code here if needed
+function debounce(func, wait) {

```

Comment:
This `debounce` function is optional. I added it for performance improvement while scrolling.  I am open to remove this function and directly call `window.addEventListener('scroll', checkHeroVisibility);` , if you prefer.